tests/selfapps_layered_e2e.ps1

```
@@ -0,0 +1,252 @@
+    }
+}
+
+$statusPath = Join-Path $workDir '~bootstrap.status.json'
+$statusExit = $null
+$statusState = $null
+if (Test-Path $statusPath) {
+    try {
+        $status = Get-Content -LiteralPath $statusPath -Raw -Encoding ASCII | ConvertFrom-Json
+        $statusExit = $status.exitCode
+        $statusState = $status.state
+    } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)